src/Reporting/MissingRectorRulesReporter.php

```diff
@@ -17,26 +16,29 @@ final class MissingRectorRulesReporter
     private $symfonyStyle;
 
     /**
-     * @var RectorInterface[]
+     * @var FileProcessorInterface[]
      */
-    private $rectors = [];
+    private $fileProcessors;
 
     /**
-     * @param RectorInterface[] $rectors
+     * @param FileProcessorInterface[] $fileProcessors
      */
-    public function __construct(array $rectors, SymfonyStyle $symfonyStyle)
+    public function __construct(SymfonyStyle $symfonyStyle, array $fileProcessors)
     {
         $this->symfonyStyle = $symfonyStyle;
-        $this->rectors = $rectors;
+        $this->fileProcessors = $fileProcessors;
     }
 
     public function reportIfMissing(): ?int
     {
-        $activeRectors = array_filter($this->rectors, function (RectorInterface $rector): bool {
-            return ! $rector instanceof PostRectorInterface;
-        });
+        $activeFileProcessors = array_filter(
+            $this->fileProcessors,
+            function (FileProcessorInterface $fileProcessor): bool {
+                return $fileProcessor->isActive();
+            }
+        );
 
-        if ($activeRectors !== []) {
+        if ($activeFileProcessors !== []) {
             return null;
         }
```

Comment:
I don't see much advantage over 3 lines above. This change requires to implement a method by every FileProcessor, that will be always the same.

Reply:
True. I have missed this. As you have done the CompilerPass change i think we can close this PR here. Everything is possible now.

Reply:
Agreed. Thanks!